tests/codegen/vec-in-place.rs

```diff
@@ -37,6 +37,8 @@ pub struct Baz {
 #[no_mangle]
 pub fn vec_iterator_cast_primitive(vec: Vec<i8>) -> Vec<u8> {
     // CHECK-NOT: loop
+    // CHECK: call
+    // CHECK-SAME: void @llvm.assume(i1 %{{.+}})
     // CHECK-NOT: call
```

Comment:
Now it only does a `CHECK-NOT: loop` before and a `CHECK-NOT: call` after the new `CHECK: call`.
Which means it allows a call before and a loop after that.
You need to duplicate the check-not blocks.